app/src/data/MetricConfig.ts

```diff
@@ -0,0 +1,133 @@
+import { VariableId } from "../data/variableProviders";
+
+export type MetricType =
+  | "count"
+  | "pct_share"
+  | "pct_share_to_pop_ratio"
+  | "per100k"
+  | "percentile"
+  | "index";
+
+export type MetricConfig = {
+  metricId: VariableId;
+  fullCardTitleName: string;
+  shortVegaLabel: string;
+  type: MetricType;
+};
+
+export type VariableConfig = {
+  variableId: string;
+  metrics: Record<string, MetricConfig>;
+};
+
+export const METRIC_CONFIG: Record<string, VariableConfig[]> = {
+  covid: [
```

Comment:
same here, let's use a type literal for this as well

Reply:
leaving TODO per our convo